briefcase-forms.rst

```diff
@@ -147,7 +147,7 @@ Export form data
 This command exports form data with id market_prices from Briefcase's storage directory on the :file:`~/Desktop` and write CSV file to :file:`~/Desktop/market_prices.csv`
 
 
-To get help about the command line operation type `java -jar path_to_jar -help`.
+To get help about the command line operation type *java -jar path_to_jar -help*.
```

Comment:
Inline references to terminal commands use `:command:`
http://docs.opendatakit.org/contributing/#role-command

(Note: The entire command, including options, etc. goes in the `:command:` role...
the `:option:` role is only for talking about options separately.)